tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -141,42 +127,54 @@ KUBETEST2_ARGS+=("--kops-binary-path=${KOPS_BIN}")
 KUBETEST2_ARGS+=("--admin-access=${ADMIN_ACCESS:-}")
 KUBETEST2_ARGS+=("--env=KOPS_FEATURE_FLAGS=${KOPS_FEATURE_FLAGS}")
 
+if [[ "${CLOUD_PROVIDER}" == "gce" ]]; then
+  KUBETEST2_ARGS+=("--gcp-project=k8s-infra-e2e-scale-project") # for dev purposes
+  KUBETEST2_ARGS+=("--stage-location=gs://k8s-infra-e2e-scale-project-stage")
+  export -n KOPS_BASE_URL
+  create_args+=("--set=spec.kubeDNS.provider=KubeDNS")
+  # print ssh key pairs
+  gcloud compute project-info describe \
+    --format="value(commonInstanceMetadata[items][ssh-keys])" \
+    --project=k8s-infra-e2e-scale-project
+  cat /etc/ssh-key-secret/ssh-public
+  mkdir -p "/tmp/kops/${CLUSTER_NAME}"
+  cp /etc/ssh-key-secret/ssh-private "/tmp/kops/${CLUSTER_NAME}/id_rsa"
+  export KUBE_SSH_KEY_PATH="/tmp/kops/${CLUSTER_NAME}/id_rsa"
+fi
+
 # More time for bigger clusters
 KUBETEST2_ARGS+=("--validation-wait=55m")
+KUBETEST2_ARGS+=("--validation-count=5") # reduce Cloud API calls
```

Comment:
```suggestion
KUBETEST2_ARGS+=("--validation-count=3") # reduce Cloud API calls
```

Reply:
ack

I want to run the entire thing via a single kubetest2 commands so we can log the build, cluster up and down, timings for cloud/kops regression

It seems cl2 is picky about something